src/app/mini-games/sprint/utils/utils.ts

```diff
@@ -0,0 +1,37 @@
+import { Sound, StreakLevel } from 'src/app/core/models/ISprintGame';
+import { SoundSrc } from 'src/app/core/constants/sprint-game';
+
+export function getRandomNumber(maxNumber: number): number {
+  return Math.floor(Math.random() * maxNumber);
+}
+
+export function isTranslationCorrect(): boolean {
+  return Math.random() >= 0.5;
+}
+
+export function getPointsMultiplier(streak: number) {
+  if (streak < StreakLevel.FIRST) return 1;
+  if (streak < StreakLevel.SECOND) return 2;
+  if (streak < StreakLevel.THIRD) return 4;
+  else return 8;
+}
+
+export function playSound(src: string) {
+  const audio = new Audio();
+  if (src === Sound.CORRECT) audio.src = SoundSrc.CORRECT;
+  else if (src === Sound.WRONG) audio.src = SoundSrc.WRONG;
+  else if (src === Sound.LEVELUP) audio.src = SoundSrc.LEVELUP;
+  else audio.src = `data:audio/mpeg;base64,${src}`;
+  audio.play();
+}
+
+export function getRandomPages() {
+  const pages: number[] = [];
+  const NUMBER_OF_PAGES = 30;
```

Comment:
@Suggestion
```NUMBER_OF_PAGES``` is common constant so it's better to move it to a common const file

Reply:
Created in constants/common.ts `NUMBER_OF_PAGES` constant.